lib/kino/smart_cell.ex

```diff
@@ -391,4 +391,12 @@ defmodule Kino.SmartCell do
     |> Inspect.Algebra.format(90)
     |> IO.iodata_to_binary()
   end
+
+  def get_shared_attr(key) do
+    Kino.SharedAttribute.get_attr(key)
+  end
+
+  def put_shared_attr(key, value) do
+    Kino.SharedAttribute.put_attr(key, value)
+  end
```

Comment:
Let's not expose, and since it's a simple delegate, we can just call these directly :)

```suggestion
```